.circleci/config.yml

```diff
@@ -85,6 +93,9 @@ jobs:
           command: |
             sudo pip install codecov
             codecov
+            ./cc-test-reporter format-coverage -t coverage.py coverage.xml
```

Comment:
Remove all codecov-related statements.

Reply:
The code-climate website where they show the coverage is not very gui friendly. For example, it does not show the miss or hit lines. So I kept it. Basically I want code-climate for just enforcing the commit status

Reply:
No, I don't think that's right -- we should be using a single service for coverage reports. 
If that's the case, I think you should try to resolve the problems with codecov's check. That might be easier/better to do here.

Reply:
Hmm I tried resolving the problems with codecov but I couldn't. I guess then I will delete the codecov statements here

Reply:
Okay, cool. Just a follow-up to this:
> The code-climate website where they show the coverage is not very gui friendly. For example, it does not show the miss or hit lines.

Will the PR author be able to know where he/she is missing the coverage?

Reply:
Unfortunately, I don't think so. I know this is gonna be a problem but I am investigating more into this

Reply:
I looked into this and found out we can look at the hit/miss lines after we finish setting up code-climate

Reply:
Could you point to a URL where the reviewers can see this?

Reply:
> Remove all codecov-related statements.

Done

Reply:
> Could you point to a URL where the reviewers can see this?

@seanlip The url is: https://codeclimate.com/github/oppia/oppia/code.
The coverage is empty right now but would be populated once we set it up.

Reply:
Ah, I meant: could you point to a sample URL where reveiwers can see what the hit/miss lines look like? Doesn't have to be the oppia repo.

Reply:
https://codeclimate.com/github/18F/calc/calc/settings.py/source. Tick the coverage field, you can see the coverage